src/components/SubMenu.vue

```diff
@@ -28,23 +37,36 @@ export default defineComponent({
       type: Object as PropType<MenuItem>,
       required: true,
     },
+    anchor: Object as PropType<QMenu['anchor']>,
```

Comment:
`anchor` and `self` would be of the same type as the `QMenu` properties with the same names.

Reply:
That's the magic, I did not see that coming at all.  Great job